target/linux/generic/pending-5.4/411-mtd-partial_eraseblock_write.patch

```diff
@@ -73,10 +73,9 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  
  	instr->addr += part->offset;
  	ret = part->parent->_erase(part->parent, instr);
-@@ -213,6 +262,24 @@ static int part_erase(struct mtd_info *m
+ 	if (instr->fail_addr != MTD_FAIL_ADDR_UNKNOWN)
  		instr->fail_addr -= part->offset;
```

Comment:
I should change this, so that if the erase fails before the write address, we avoid the write?